src/mlpack/methods/linear_svm/linear_svm.hpp

```diff
@@ -202,13 +202,15 @@ class LinearSVM
    * @param numClasses Number of classes for classification.
    * @param lambda L2-regularization constant.
    * @param optimizer Desired optimizer.
+   * @param callbacks Callback Functions.
    * @return Objective value of the final point.
    */
-  template <typename OptimizerType = ens::L_BFGS>
+  template <typename OptimizerType = ens::L_BFGS, typename... CallbackTypes>
   double Train(const MatType& data,
                const arma::Row<size_t>& labels,
                const size_t numClasses = 2,
-               OptimizerType optimizer = OptimizerType());
+               OptimizerType optimizer = OptimizerType(),
+               CallbackTypes&&... callbacks);
```

Comment:
Can you add a description for the template parameter as well? See https://github.com/mlpack/mlpack/blob/db03fbf4838a4e6226f0f332857528ab195bf4ec/src/mlpack/methods/linear_svm/linear_svm.hpp#L199 for en example.